includes/class-ssp-frontend.php

```diff
@@ -340,31 +421,16 @@ public function media_player ( $srcFile = '', $episode_id = 0 ) {
 						$thumb_id = get_post_thumbnail_id( $episode_id );
 
 						if ( ! empty( $thumb_id ) ) {
-							list( $src, $width, $height ) = wp_get_attachment_image_src( $thumb_id, 'full' );
-							$albumArt = compact( 'src', 'width', 'height' );
-						} else {
 
-							// First fall back to series image, and then finally a no album art image
-							$series_id = false;
+						    list( $src, $width, $height ) = wp_get_attachment_image_src( $thumb_id, 'full' );
 
-							if( $series = get_the_terms( $episode_id, 'series' ) ){
-								$series_id = ( !empty( $series ) && isset( $series[0] ) ) ? $series[0]->term_id : false;
-							}
+							$albumArt = compact( 'src', 'width', 'height' );
 
-                            if( $series_id && $series_image = get_option( "ss_podcasting_data_image_{$series_id}" ) ){
-                                $series_image_attachment_id = ssp_get_image_id_from_url( $series_image );
-                                list( $src, $width, $height ) = wp_get_attachment_image_src( $series_image_attachment_id, 'medium' );
-                                $albumArt = compact( 'src', 'width', 'height' );
-                            }elseif( $series_image = get_option( "ss_podcasting_data_image" ) ){
-	                            $series_image_attachment_id = ssp_get_image_id_from_url( $series_image );
-	                            list( $src, $width, $height ) = wp_get_attachment_image_src( $series_image_attachment_id, 'medium' );
-	                            $albumArt = compact( 'src', 'width', 'height' );
-                            }else{
-                                $albumArt['src'] = SSP_PLUGIN_URL . '/assets/images/no-album-art.png';
-                                $albumArt['width'] = 300;
-                                $albumArt['height'] = 300;
```

Comment:
First check for album art in the form of a featured image, and if not square (or not present) proceed to use the new album art method to get the cover image.